src/components/search-box.vue

```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
     <SearchInput
       :autofocus="false"
       :placeholder="$t('searchBox.placeholder')"
-      :class="$x.device === 'mobile' ? 'x-padding--left-05' : 'x-padding--left-08'"
+      :instant="false"
+      class="desktop:x-pl-40"
     />
 
-    <!-- TODO: Remove x-max-h when implementing XDS input-group -->
     <ClearSearchInput
       v-if="$x.device === 'desktop' && $x.query.searchBox"
-      class="x-button-lead !x-button-sm x-button-ghost x-max-h-32"
+      class="x-input-group-button"
     >
       {{ $t('searchBox.clear') }}
     </ClearSearchInput>
 
-    <button
+    <SearchButton
```

Comment:
The search button is now bigger than the one in the `home-template` so the same as above with the font

Reply:
I don't see it bigger. It is 40px x 40px in both cases. Also the icon is 24px in both cases